blaze/server/server.py

```diff
@@ -342,18 +346,56 @@ def compserver(serial_format):
         result = compute(expr, {leaf: dataset})
 
         if iscollection(expr.dshape):
-            result = odo(result, list)
+            result = odo(
+                result,
+                chunks(list) if chunksize > 0 else list,
```

Comment:
if you used `None` then you can say `if chunksize else list` and the same for when you call it in the `/compute.<format>` route